src/App.js

```diff
@@ -1,16 +1,87 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList'
+import TodoForm from './components/TodoComponents/TodoForm'
+import './reset.css'
+import './index.css'
+//import './BobRoss.react'
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  // this is your world, you're the creator, find freedom on this canvas
+  constructor() {
+    super()
+    this.state = {
+      items: [],
+      value: '',
+      completed: false,
+    }
+  }
+  componentDidMount() {
+    // believe that you can do it cause you can do it
+    if(localStorage.hasOwnProperty('items')) {
+      // we don't load mistakes, just happy little accidents
+      this.setState({items: JSON.parse(localStorage.getItem('items'))})
+    }
+  }
+  toggleComplete = target => {
+    // let's put a nice big strong x on the list
+    let items = this.state.items
```

Comment:
you could use desctructuring here and make this simply const { item } = this.state

less typing and easier to read